lib/bootloader_setup.pm

```diff
@@ -465,7 +465,12 @@ sub bootmenu_default_params {
         push @params, get_hyperv_fb_video_resolution;
         push @params, 'namescheme=by-label' unless is_jeos or is_caasp;
     }
-    type_string_very_slow(" @params ");
+    if (get_var('TYPE_BOOT_PARAMS_FAST')) {
+        type_string(" @params ");
+    }
+    else {
+        type_string_very_slow(" @params ");
```

Comment:
Instead of many if-else, how about renaming `type_string_very_slow` to type_slow_if_needed and handling inside there?

What you could also do is to set the typing speed in general based on a test variable

Reply:
Option A
> Instead of many if-else, how about renaming `type_string_very_slow` to type_slow_if_needed and handling inside there?

Option B
> What you could also do is to set the typing speed in general based on a test variable

I prefer option B